### PR TITLE
Update DockerFile for M1 Macs

### PR DESCRIPTION
**Pull request by kaiobrito**
### Changed
- Update Docker dependencies for M1 Macs

---
The DockerFile compiles by using apt-get to install the dependencies instead of installing them manually. Unfortunately, the PostGIS docker image isn't fully compatible with M1 chips, which may be related to https://github.com/postgis/docker-postgis/issues/216. On my machine, I was able to run the migrations and fixtures but after some usage, the DB restarts and can't recover. 

The PR is still valid it won't break on Intel PCs and as soon as a new PostGIS is stable we can pull the latest version of the image.

### CODE REVIEW

File: build_scripts/docker-setup.sh [hunk 1]
```diff
@@ -23,45 +23,14 @@ export NODE_VERSION=10.15.0
...
-
-curl -SLO "https://nodejs.org/dist/v${NODE_VERSION}/node-v${NODE_VERSION}-linux-x64.tar.xz"
-curl -SLO "https://nodejs.org/dist/v${NODE_VERSION}/SHASUMS256.txt.asc"
-gpg --batch --decrypt --output SHASUMS256.txt SHASUMS256.txt.asc
-grep " node-v${NODE_VERSION}-linux-x64.tar.xz\$" SHASUMS256.txt | sha256sum -c -
-tar -xJf "node-v${NODE_VERSION}-linux-x64.tar.xz" -C /usr/local --strip-components=1
-rm "node-v${NODE_VERSION}-linux-x64.tar.xz" SHASUMS256.txt.asc SHASUMS256.txt
-
+# install nodejs, taken from https://github.com/nodesource/distributions/blob/master/README.md
+curl -fsSL https://deb.nodesource.com/setup_10.x | bash -
```
Comment (madewulf) [inline build_scripts/docker-setup.sh:32]: I don't know this website for installing node. How can I be sure that this is not introducing a security problem?

**Review by madewulf — COMMENTED** [review]
I don't totally understand: is our docker installation running on M1 after this or not?

File: build_scripts/docker-setup.sh (hunk 1 above)
Comment (kaiobrito) [reply build_scripts/docker-setup.sh:32]: It's referenced in the Nodejs website https://nodejs.org/en/download/package-manager/#debian-and-ubuntu-based-linux-distributions

**Comment by kaiobrito** [comment]
> I don't totally understand: is our docker installation running on M1 after this or not?

Partially! These changes are needed so the M1 chips can build and start the `iaso` container. However, the Postgis database doesn't support the M1 chips yet and crashes after the web app's login.

**Comment by vinch** [comment]
@kaiobrito I have the following error when doing `docker-compose up db` on my machine:

```
Starting iaso_db_1 ... done
Attaching to iaso_db_1
db_1       |
db_1       | PostgreSQL Database directory appears to contain a database; Skipping initialization
db_1       |
db_1       | 2021-04-16 07:53:53.396 UTC [1] FATAL:  database files are incompatible with server
db_1       | 2021-04-16 07:53:53.396 UTC [1] DETAIL:  The data directory was initialized by PostgreSQL version 10, which is not compatible with this version 12.6 (Debian 12.6-1.pgdg100+1).
iaso_db_1 exited with code 1
```

**Review by edarchis — COMMENTED** [review]
The upgrade of PG in Docker is 🤮. Dump DB, delete, reload:
https://josepostiga.com/how-to-upgrade-postgresql-version-and-transfer-your-old-data-using-docker/

I am testing the upgrade on AWS at the moment. I'll update when it's done.

**Pull request merged**